ultraplot/figure.py

```diff
@@ -824,6 +824,10 @@ def _parse_backend(backend=None, basemap=None):
         Handle deprecation of basemap and cartopy package.
         """
         # Basemap is currently being developed again so are removing the deprecation warning
+        if backend == "basemap":
+            warnings._warn_ultraplot(
+                f"{backend=} will be deprecated in next major release (v1.6)"
```

Comment:
Please be careful about version numbers. Version 1.6 < 1.50 which is where we are now. So this needs to be 1.60. Also, a "major release" would be `2.*`. What do you mean here?